extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -240,9 +240,17 @@ export class AmpGeo extends AMP.BaseElement {
 
     return this.mode_ !== mode.GEO_API
       ? Promise.resolve()
-      : this.fetchCountry_().then((country) => {
-          if (country) {
-            this.country_ = country;
+      : this.fetchCountry_().then((data) => {
+          if (data) {
+            // Country is required and guaranteed to exist if data is available.
+            this.country_ = data.country;
```

Comment:
Nit: I would destructure into local variables, to save bytes. (My personal preference it to avoid multiple `obj.foo` references as much as possible.)